assets/export/docx/7x10/word/document.xml

```diff
@@ -0,0 +1,912 @@
+<?xml version="1.0" encoding="utf-8"?>
+<w:document xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main"
+xmlns:m="http://schemas.openxmlformats.org/officeDocument/2006/math"
+xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships"
+xmlns:o="urn:schemas-microsoft-com:office:office"
+xmlns:v="urn:schemas-microsoft-com:vml"
+xmlns:w10="urn:schemas-microsoft-com:office:word"
+xmlns:a="http://schemas.openxmlformats.org/drawingml/2006/main"
+xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture"
+xmlns:wp="http://schemas.openxmlformats.org/drawingml/2006/wordprocessingDrawing">
+
+  <w:body>
+    <w:p>
+        <w:pPr>
+            <w:pStyle w:val="Head Separator" />
+<!--            <w:sectPr>
```

Comment:
to remove?

Reply:
I think it should stay (and probably be updated to include its sibling section breaks) but I'll write a comment about it to explain that.

Reply:
I did not get to documenting this better. I figured I just needed to move forward— but it'll be a good 'in-between' project of bite-sized updates.